csa-runner-app/src/test/java/com/farao_community/farao/swe_csa/app/dichotomy/SweCsaDichotomyRunnerTest.java

```diff
@@ -84,22 +85,11 @@ public SweCsaRaoValidatorMock(FileExporter fileExporter, AsynchronousRaoRunnerCl
         }
 
         @Override
-        public DichotomyStepResult validateNetwork(String stepFolder, Network network, Crac crac, CsaRequest csaRequest, String raoParametersUrl, boolean withVoltageMonitoring, boolean withAngleMonitoring, CounterTradingValues counterTradingValues) {
-            RaoParameters raoParameters = new RaoParameters();
-            JsonRaoParameters.update(raoParameters, getClass().getResourceAsStream("/RaoParameters.json"));
-            Network networkCopy = NetworkSerDe.copy(network);
-            RaoInput raoInput = RaoInput.build(networkCopy, crac).build();
-            RaoResult raoResult = FastRao.launchFilteredRao(raoInput, raoParameters, null, criticalCnecs);
-
+        public DichotomyStepResult validateNetwork(Network network, Crac crac, CsaRequest csaRequest, String raoParametersUrl, boolean withVoltageMonitoring, boolean withAngleMonitoring, CounterTradingValues counterTradingValues) {
             RaoResponse raoResponse = Mockito.mock(RaoResponse.class);
-            Set<FlowCnec> frEsFlowCnecs = crac.getFlowCnecs().stream()
-                .filter(flowCnec -> flowCnec.isOptimized() && flowCnec.getLocation(network).contains(Optional.of(Country.FR)))
-                .collect(Collectors.toSet());
-            Set<FlowCnec> ptEsFlowCnecs = crac.getFlowCnecs().stream()
-                .filter(flowCnec -> flowCnec.isOptimized() && flowCnec.getLocation(network).contains(Optional.of(Country.PT)))
-                .collect(Collectors.toSet());
-            boolean cnecsOnPtEsBorderAreSecure = hasNoFlowCnecNegativeMargin(raoResult, ptEsFlowCnecs);
-            boolean cnecsOnFrEsBorderAreSecure = hasNoFlowCnecNegativeMargin(raoResult, frEsFlowCnecs);
+            RaoResult raoResult = Mockito.mock(RaoResult.class);
+            boolean cnecsOnPtEsBorderAreSecure = counterTradingValues.getPtEsCt() >= 0 && counterTradingValues.getPtEsCt() < 50;
+            boolean cnecsOnFrEsBorderAreSecure = counterTradingValues.getFrEsCt() >= 600 && counterTradingValues.getFrEsCt() <= 2012;
```

Comment:
if you want the test to be more representative of reality:
```suggestion
            boolean cnecsOnPtEsBorderAreSecure = counterTradingValues.getPtEsCt() >= 0;
            boolean cnecsOnFrEsBorderAreSecure = counterTradingValues.getFrEsCt() >= 600;
```